### DISCUSSION TITLE
Listing all direct dependencies of a workspace

### DISCUSSION THREAD

**Discussion by zkochan** [pnpm/pnpm#5594] [Ideas]
Currently if I want to list all dependencies in a workspace, I can run: `pnpm list -r`. However, this will print the same dependencies many times, each time it is used in a workspace project. It would be nice to have a command that would list all unique direct dependencies. Similar to how `pnpm outdate -r` works but it would list not outdated deps but all deps.

**Comment by weyert** [comment]
Maybe we could reuse the code from licenses for this?
